### DISCUSSION TITLE
mysql error in expense app

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1808] [Q&A] [answered]
![Screenshot (21)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/b26903e1-372c-48df-9d0f-4fc4a9c45a23)

**Comment by chaithanya1812** [comment, marked as answer]
Hello, Bhashitha,
pls Setup the MySQL5.7 repo file
follow this
commnad:-       vi /etc/yum.repos.d/mysql.repo
 
And add content & save it.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1b5554c0-4839-4559-8910-8a41f910d610)
